src/components/InputTypeCountry.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { countries } from 'country-data';
+
+import StyledSelect from './StyledSelect';
+
+class InputTypeCountry extends Component {
+  static propTypes = {
+    name: PropTypes.string,
+    onChange: PropTypes.func,
+    defaultValue: PropTypes.string,
+  };
+
+  state = {
+    error: false,
+  };
+
+  handleChange = ({ value }) => {
+    if (value === null) {
+      this.setState({
+        error: true,
+      });
+    } else {
+      this.setState({
+        error: false,
+      });
+      this.props.onChange(value);
+    }
+  };
+
+  getCountryName(countryISO) {
+    const { name } = countries[countryISO];
```

Comment:
This crashes when I use an account that doesn't have a country. `defaultValue` will be null for all existing users when we deploy, we need to be more defensive here.